build_tools/travis/install.sh

```diff
@@ -48,6 +48,7 @@ popd
 source activate testenv
 
  # Install requirements via pip in our conda environment
+conda install -y pytorch cpuonly -c pytorch-nightly
```

Comment:
Interestingly if installed via `conda`, Python3.5 fails with the following error message;  ([link](https://travis-ci.org/github/pytorch/audio/jobs/674611278#L642))

```
Requirement already satisfied: torch>=1.4.0 in /home/travis/miniconda3/envs/testenv/lib/python3.5/site-packages (from -r requirements.txt (line 1)) (1.6.0.dev20200413)
torch requires Python '>=3.6.1' but the running Python is 3.5.6
```

NB: Using `pip` to install nightly works but it is not really a nightly.
`pip install --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html`

```
pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
Looking in indexes: https://pypi.org/simple, http://100.97.64.150
Looking in links: https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
Collecting torch
  Downloading https://download.pytorch.org/whl/nightly/cpu/torch-1.5.0%2Bcpu-cp38-cp38-linux_x86_64.whl (127.2 MB)
     |████████████████████████████████| 127.2 MB 63.9 MB/s
Requirement already satisfied: numpy in /private/home/moto/miniconda3/envs/testenv/lib/python3.8/site-packages (from torch) (1.18.2)
Installing collected packages: torch
Successfully installed torch-1.5.0+cpu
```